x-pack/legacy/plugins/monitoring/public/views/no_data/index.js

```diff
@@ -7,24 +7,9 @@
 import uiRoutes from 'ui/routes';
 import template from './index.html';
 import { NoDataController } from './controller';
-import { CODE_PATH_LICENSE } from '../../../common/constants';
+
 uiRoutes
   .when('/no-data', {
     template,
-    resolve: {
```

Comment:
What's the thinking behind this page? I'm guessing it's to avoid doing the check that we might fail, but we still should be doing this check so we can do the redirect, if necessary.

Reply:
The redirect logic will still happen here: https://github.com/elastic/kibana/pull/43338/files#diff-87ec11d43f1e502792408786cf726eb6R96

We should avoid doing any kinds of requests in `resolve`, since it provides no template in which we can indicate a loading state to the user.